e2e/components/SelectPanel.test.ts

```diff
@@ -112,7 +113,9 @@ test.describe('SelectPanel', () => {
   test(`Default @vrt responsive width .modern-action-list--true`, async ({page}) => {
     await visit(page, {
       id: 'components-selectpanel--default',
-      globals: {featureFlags: {primer_react_select_panel_with_modern_action_list: true}},
+      globals: {
+        featureFlags: {primer_react_select_panel_with_modern_action_list: true, primer_react_css_modules_ga: true},
+      },
```

Comment:
is there a css modulers related regression here? 👀 , why is this one needed?

Reply:
It's needed because i removed the styles from sx (your other comment), so it doesn't work without css modules enabled :)

Reply:
Fixed :)